steps/graph.py

```diff
@@ -9,10 +12,10 @@
 
 
 def generate_random_graph_erdos_renyi(
-    number_of_nodes: int,
-    edge_probability: float,
-    random_weights: bool = False,
-    seed: Optional[int] = None,
+        number_of_nodes: int,
```

Comment:
Looks to me like that's too much indentation.

Reply:
I think PEP wants an additional indentation when there is a line break.

Reply:
Fine 👍